tests/property/ic_api/arg_data_raw/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "scripts": {
+        "pretest": "tsx test/pretest.ts",
+        "test": "jest"
+    },
+    "dependencies": {
+        "azle": "0.24.1"
+    },
+    "devDependencies": {
```

Comment:
See if you can remove the devDependencies...I am assuming those dependencies will then be taken from Azle. If this works for repo and npm tests...maybe we should remove these devDependencies in all tests and examples?

Reply:
done

Reply:
Let's see if we can remove all dev dependencies

Reply:
we can get all but jest_ts

Reply:
attempting ts-jest